roles/foreman_provisioning/meta/default.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - role: hammer_creadentials
```

Comment:
s/hammer_creadentials/hammer_credentials/

This is also a new dependency and I'd prefer to use ```foreman::cli``` through the installer since we don't have to replicate things then.